Dockerfile.localdev

```diff
@@ -0,0 +1,30 @@
+FROM rust:1.80-alpine3.20 as builder
```

Comment:
since errors are different by platform, made this local docker file so we can quickly develop on linux. i'll add a comment at the top with instructions on how to use